tbx/project_styleguide/templates/patterns/navigation/components/breadcrumbs.html

```diff
@@ -1,25 +1,23 @@
 {% load wagtailcore_tags %}
 
-{% if page.get_ancestors|length > 1 %}
+{% if page.breadcrumbs %}
     <div class="grid">
         <nav aria-label="breadcrumb" class="breadcrumbs-nav">
             <ul class="breadcrumbs-nav__list">
-                {% with parent_page=page.get_parent %}
-                    {% for ancestor_page in page.get_ancestors %}
-                        {% if not ancestor_page.is_root %}
-                            <li class="breadcrumbs-nav__item {% if ancestor_page == parent_page or ancestor_page.pattern_library_is_parent %}breadcrumbs-nav__item--parent{% else %}breadcrumbs-nav__item--ancestor{% endif %}">
```

Comment:
The `{% if ancestor_page == parent_page %}` didn't actually work (it's always false) so I switched to using `{% if forloop.last %}` because the direct parent should always be the last element of the ancestors.